tests/tests-openshift-template.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: jenkins-on-openshift-testing
+objects:
+
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ansible
+    labels:
+      role: jenkins-slave
+
+
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: ansible
+  spec:
+    strategy:
+      type: Docker
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ansible:latest
+    source:
+      dockerfile: |
+        FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+        USER root
+
+        RUN yum clean all && \
+            yum-config-manager --enable rhel-7-server-rpms && \
+            yum-config-manager --enable rhel-7-server-extras-rpms && \
+            yum-config-manager --enable rhel-7-server-ose-3.7-rpms && \
+            yum-config-manager --enable rhel-7-fast-datapath-rpms && \
+            INSTALL_PKGS="atomic-openshift-utils" && \
+            yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+            rpm -V $INSTALL_PKGS && \
+            yum clean all
+        USER 1001
+      type: Docker
+    triggers:
+    - imageChange: {}
+      type: ImageChange
```

Comment:
I think we want a `ConfigChange` trigger as well... and/or document that a build of ansible should be performed before running the test pipeline